.github/workflows/run-test/action.yml

```diff
@@ -53,11 +53,16 @@ runs:
         MODE: ${{ inputs.mode }}
         SCOPE: ${{ inputs.scope }}
         STACK_VERSION: ${{ inputs.stack-version }}
+    - uses: gacts/github-slug@7ef5f322bcd0902e63d75aae219a71367852a7b9 # 1.3.1
```

Comment:
do you mind adding a comment in the description explaining why?

Reply:
```suggestion
    # The upload-artifact action fails if the name contains slashes `/` or colons `:`.
    # This step creates a slugified version that will not contain these characters.
    - uses: gacts/github-slug@7ef5f322bcd0902e63d75aae219a71367852a7b9 # 1.3.1
```